There's a  CI failure in the Python postgres unit tests but it  appears to be unrelated to my changes. 
The failing test is:
```
FAILED kolibri/core/content/test/test_content_app.py::ContentNodeAPITestCase::test_contentnode_tree
This is a backend content API test, while my PR only modifies frontend test files in the Device plugin. This looks like it may be a flaky test.
```

could a maintainer re-run the failed check when possible? 
and let me know if any other changes need to be made. Thanks.

--- a/kolibri/plugins/device/frontend/views/DeviceSettingsPage/__tests__/DeviceSettingsPage.spec.js
+++ b/kolibri/plugins/device/frontend/views/DeviceSettingsPage/__tests__/DeviceSettingsPage.spec.js
@@ -142,7 +142,7 @@ describe('DeviceSettingsPage', () => {
     expect(allowGuestAccess).not.toBeChecked();
     expect(disallowGuestAccess).not.toBeChecked();
     expect(unlistedChannels).toBeChecked();
-    expect(screen.getAllByText('English')[0]).toBeInTheDocument();
+    expect(screen.getByTestId('languageSelect')).toHaveTextContent('English');
   });
 
   function setMockedData(allowGuestAccess, allowAllAccess, allowPeerUnlistedChannelImport = false) {

--- a/kolibri/plugins/device/frontend/views/DeviceSettingsPage/index.vue
+++ b/kolibri/plugins/device/frontend/views/DeviceSettingsPage/index.vue
@@ -35,6 +35,7 @@
             :disabled="language.value === undefined"
             :floatingLabel="false"
             style="max-width: 300px"
+            data-testid="languageSelect"
           />
         </div>
 

